vra/resource_flavor_profile.go

```diff
@@ -34,7 +34,7 @@ func resourceFlavorProfile() *schema.Resource {
 				Optional: true,
 			},
 			"external_region_id": {
-				Type:     schema.TypeMap,
+				Type:     schema.TypeString,
```

Comment:
Can you verify this in the data source for flavor profile as well?

Reply:
There is no such data source.

Reply:
okay, that's a miss. We will have to add one.